### PR TITLE
issue-18442-update-enterprise-screens-in-legacy-portlet fix

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
We need to update the old "get Enterprise" messages in the different screens in the system. You can see the old messages in community edition here:

Time Machine
ESSearch
Workflows
Rules
Publishing queue

### CODE REVIEW

File: dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/network/not_licensed.jsp [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
+    .request-license-button { background: var(--color-main); border: solid 1px transparent; border-radius: 2px; color: #fff; display: inline-block; line-height: 36px; padding: 0 24px; text-decoration: none; text-transform: uppercase; }
+    .request-license-button:hover { background: var(--color-main_mod); border: solid 1px transparent; }
 </style>
-
-<div class="wrapperCluster">
-	<div class="content">
-		<h2><%=LanguageUtil.get(pageContext, "com.dotcms.repackage.javax.portlet.title.EXT_CLUSTERING_TOOL")%></h2>
-		<p><%= LanguageUtil.get(pageContext, "CLUSTERING-NOT-LICENSED") %></p>
-	</div>
+<div class="portlet-wrapper">
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/network/not_licensed.jsp:15]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5641121)

File: dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/publishing/not_licensed.jsp [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
+    .request-license-button { background: var(--color-main); border: solid 1px transparent; border-radius: 2px; color: #fff; display: inline-block; line-height: 36px; padding: 0 24px; text-decoration: none; text-transform: uppercase; }
+    .request-license-button:hover { background: var(--color-main_mod); border: solid 1px transparent; }
 </style>
-
-<div class="wrapperPublishing">
-	<div class="content">
-		<h2><%=LanguageUtil.get(pageContext, "publisher_Publishing_Environment")%></h2>
-		<p><%= LanguageUtil.get(pageContext, "PUBLISHING-NOT-LICENSED") %></p>
-	</div>
+<div class="portlet-wrapper">
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/publishing/not_licensed.jsp:15]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5641121)

File: dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/network/not_licensed.jsp [hunk 3]
```diff
@@ -1,19 +1,32 @@
...
+    <div class="unlicense-content">
+        <i class="material-icons" style="font-size: 120px;">storage</i>
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/network/not_licensed.jsp:17]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5641121)

File: dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/publishing/not_licensed.jsp [hunk 4]
```diff
@@ -1,19 +1,32 @@
...
+    <div class="unlicense-content">
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/publishing/not_licensed.jsp:16]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5641121)

File: dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/network/not_licensed.jsp (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/network/not_licensed.jsp:17]: Codacy Issue found: [Avoid having style information in JSP files.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5641121)

File: dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/network/not_licensed.jsp [hunk 5]
```diff
@@ -1,19 +1,32 @@
...
+        <ul>
+            <li>
+                <a target="_blank" href="https://dotcms.com/product/features/feature-list"><%=LanguageUtil.get(pageContext, "Learn-more-about-dotCMS-Enterprise")%></a>
+            </li>
+            <li>
+                <a target="_blank" href="https://dotcms.com/contact-us/"><%=LanguageUtil.get(pageContext, "Contact-Us-for-more-Information")%></a>
+            </li>
+        </ul>
+        <p style="display: block;">
+            <a class="request-license-button" href="https://dotcms.com/licensing/request-a-license-3/index"><%=LanguageUtil.get(pageContext, "request-trial-license")%></a>
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/network/not_licensed.jsp:29]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5641121)

File: dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/publishing/not_licensed.jsp [hunk 6]
```diff
@@ -1,19 +1,32 @@
...
+        <i class="material-icons" style="font-size: 120px;">cloud_upload</i>
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/publishing/not_licensed.jsp:17]: Codacy Issue found: [Avoid having style information in JSP files.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5641121)

File: dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/network/not_licensed.jsp [hunk 7]
```diff
@@ -1,19 +1,32 @@
...
+    .request-license-button:hover { background: var(--color-main_mod); border: solid 1px transparent; }
 </style>
-
-<div class="wrapperCluster">
-	<div class="content">
-		<h2><%=LanguageUtil.get(pageContext, "com.dotcms.repackage.javax.portlet.title.EXT_CLUSTERING_TOOL")%></h2>
-		<p><%= LanguageUtil.get(pageContext, "CLUSTERING-NOT-LICENSED") %></p>
-	</div>
+<div class="portlet-wrapper">
+    <div class="unlicense-content">
```
Comment (dev-dotcms) [inline dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/network/not_licensed.jsp:16]: Codacy Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5641121)

**Review by nollymar — APPROVED** [review]

**Review by freddyucv — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

**Pull request merged**